package-lock.json

```diff
@@ -2975,7 +2975,6 @@
 			"resolved": "https://registry.npmjs.org/align-text/-/align-text-0.1.4.tgz",
 			"integrity": "sha1-DNkKVhCT810KmSVsIrcGlDP60Rc=",
 			"dev": true,
-			"optional": true,
```

Comment:
I see unrelated lock file changes in this PR. My bet is that it is because this file was regenerated using npm 6.7 which introduced some changes in a way `optional` flag is handled. Can you try to run `npm install` with the latest version of npm?

Reply:
Which version did you mean? I had npm@6.1.0 when those `"optional": true,` appeared. Tried to update to the latest (`npm install -g npm@latest`) but the latest version is npm@6.7.0.

Reply:
> the latest version is npm@6.7.0

This one

Reply:
My bad, I misunderstood your comment.